kolibri/utils/logger.py

```diff
@@ -245,6 +255,11 @@ def get_default_logging_config(LOG_ROOT, debug=False, debug_database=False):
                 "level": DEFAULT_LEVEL,
                 "propagate": False,
             },
+            "cherrypy.access": {
+                "handlers": REQUEST_HANDLERS,
```

Comment:
I think we could simplify this by just doing:
`[] if DISABLE_REQUEST_LOGGING else DEFAULT_HANDLERS`

That keeps the change to the minimum required to resolve the issue I think?

Reply:
Yes. Done